pkgs/applications/office/paperless/python-modules/default.nix

```diff
@@ -0,0 +1,34 @@
+pyPkgs: fetchFromGitHub:
+{
+  django_2_0 = pyPkgs.django_2_2.overrideDerivation (_: rec {
```

Comment:
overrideDerivation should be avoided, please use `overrideAttrs`, or in the case of python packages, `overridePythonAttrs`